lib/nostrum/struct/emoji.ex

```diff
@@ -61,7 +60,6 @@ defmodule Nostrum.Struct.Emoji do
   # Unicode Emojis
   "≡ƒæì"
   "\xF0\x9F\x98\x81"
-  URI.encode("\u2b50")
```

Comment:
Sorry if I wasn't clear. Could we still have the example here just without the `URI.encode` function call?

Reply:
Oops. Fixed in latest commit.